mozilla/bedrock
# [fix bug 1254309] - Update Fx friends signup

## Description
Instead of linking off to [friends.mozilla.org](https://friends.mozilla.org/welcome), we instead want to present a newsletter subscribe form on [mozilla.org/contribute/friends/](https://www.mozilla.org/contribute/friends/).

## Bugzilla
[bug 1254309](https://bugzilla.mozilla.org/show_bug.cgi?id=1254309)

## Testing
- Click the "Sign Me Up" button in the header area to reveal the sign up form
- Complete the sign up form and ensure thank you message is displayed
- Check no-JS and RTL

## Checklist

Waiting on new newsletter to be created ([bug 1254685](https://bugzilla.mozilla.org/show_bug.cgi?id=1254685)) and for GA to be verified ([bug 1254368](https://bugzilla.mozilla.org/show_bug.cgi?id=1254368)).
